### PR TITLE
Pgadmin macros

### PR DESCRIPTION
**Pull request by gabrielwol**
functions meant to be used as PGadmin macros.

Files: 
~/pgadmin_macros/function_get_column_info_schema.sql
~/pgadmin_macros/function_get_column_info_table.sql
~/pgadmin_macros/function_get_row_count.sql

### CODE REVIEW

File: pgadmin_macros/function_get_column_info_schema.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
+--for testing: 
+--rescu
+
+DROP FUNCTION get_column_info_schema(text);
+CREATE OR REPLACE FUNCTION get_column_info_schema(IN sch_name text) --$SELECTION$ = schema_name
```
Comment (chmnata) [inline pgadmin_macros/function_get_column_info_schema.sql:5]: If its gonna live in `public` we might have to specific the schema first

File: pgadmin_macros/function_get_row_count.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+--for testing: 
+--rescu.volumes_15min
+
+DROP FUNCTION get_row_count(text);
+CREATE OR REPLACE FUNCTION get_row_count(IN schema_table text)
+RETURNS TEXT AS
+$$
+DECLARE
+  row_count TEXT;
```
Comment (chmnata) [inline pgadmin_macros/function_get_row_count.sql:9]: could it be an int?

**Review by chmnata — CHANGES_REQUESTED** [review]
Looks goooooood, some minor comments @gabrielwol

File: pgadmin_macros/function_get_row_count.sql (hunk 2 above)
Comment (gabrielwol) [reply pgadmin_macros/function_get_row_count.sql:9]: I changed it to a comma formatted text cause I'm always bothered counting the digits

**Pull request merged**